CUDADataFormats/Track/interface/PixelTrackHeterogeneous.h

```diff
@@ -21,11 +21,14 @@ class TrackSoAT {
 
   // Always check quality is at least loose!
   // CUDA does not support enums  in __lgc ...
-  eigenSoA::ScalarSoA<uint8_t, S> m_quality;
-  constexpr Quality quality(int32_t i) const { return (Quality)(m_quality(i)); }
-  constexpr Quality &quality(int32_t i) { return (Quality &)(m_quality(i)); }
-  constexpr Quality const *qualityData() const { return (Quality const *)(m_quality.data()); }
-  constexpr Quality *qualityData() { return (Quality *)(m_quality.data()); }
+  private: 
+  eigenSoA::ScalarSoA<uint8_t, S> quality_;
```

Comment:
why changing from m_ that is absolutely legit in CMS?

Reply:
Indeed, it is the "preferred" method for distinguishing member variables (rule 2.9). Slava requested the change in a few places though, and I would tend to agree that a single convention is useful (we also had local variables with trailing underscores, which was confusing (fixed)).